src/components/RichNote/hooks/loadAllMediaLinks.ts

```diff
@@ -8,6 +8,7 @@ const supportedContentTypes = [
   'image/png',
   'image/svg',
   MP4_FILE_TYPE,
+  MOV_FILE_TYPE,
```

Comment:
**suggestion** could we have a single constant for VIDEO_FILE_TYPES and use that around the place? bit of duplication here

Reply:
Ye yup gr8 minds think alike, did that earlier